src/WhiteLibrary/__init__.py

```diff
@@ -23,6 +23,7 @@
                                          TextBoxKeywords,
                                          UiItemKeywords)   # noqa: E402
 from WhiteLibrary.keywords.robotlibcore import DynamicCore   # noqa: E402
+from WhiteLibrary.errors import ItemNotFoundError   # noqa: E402
```

Comment:
noqa E402 is not necessary here because the import is actually used in line 208 ..

Reply:
E402 is "Module level import not at top of file", and flake8 will complain if the noqa is not there. I put the line with the other WhiteLibrary imports for clarity (and the keyword imports from WhiteLib module won't work if they are on top of the file).

Reply:
ack, my memory messed up - you are correct.